gme/Music_Emu.cpp

```diff
@@ -241,10 +248,22 @@ blargg_err_t Music_Emu::seek( long msec )
 	return seek_samples( msec_to_samples( msec ) );
 }
 
+blargg_err_t Music_Emu::seek_scaled(long msec)
+{
+	require(tempo_ > 0);
+	blargg_long frames = (msec / 1000.0) * sample_rate();
+	if (frames < out_time_scaled)
```

Comment:
Please, ensure code style is same (any function calls and conditions in THIS project require spaces between parens and content)